docs(auth, facebook): update fbsdk link to fbsdk-next repo (#5450)
invertase/react-native-firebase: docs/auth/social-auth.md

## Facebook

> Facebook provide an [official React Native library](https://www.npmjs.com/package/react-native-fbsdk-next) which wraps around

Reviewer: ```suggestion
There is a [community-supported React Native library](https://github.com/thebergamo/react-native-fbsdk-next) which wraps around
```
